src/interface_r/README.md

```diff
@@ -1,32 +1,142 @@
+This directory contains the R package for **H2O4GPU**, a collection of GPU solvers by [H2O.ai](https://www.h2o.ai/) with APIs in Python and R.  The Python API builds upon the easy-to-use [scikit-learn](http://scikit-learn.org) API.  The **h2o4gpu** R package is a wrapper around the **h2o4gpu** Python module.  The R package makes use of RStudio's [reticulate](https://rstudio.github.io/reticulate/) package for facilitating access to Python libraries through R.
 
-### Installation
+## Installation
 
-First, please follow the instruction [here](https://github.com/h2oai/h2o4gpu#installation) to build the H2O4GPU Python package.
+There are a few [system requirements](https://github.com/h2oai/h2o4gpu#requirements), including Ubuntu 16.04+, Python >=3.6, R >=3.1, CUDA 8 or 9, and a machine with Nvidia GPUs.  The code should still run if you have CPUs, but it will fall back to scikit-learn CPU based versions of the algorithms.
 
-Then install `h2o4gpu` R package via the following:
+The **h2o4gpu** Python module is a prerequisite for the R package. So first, follow the instructions [here](https://github.com/h2oai/h2o4gpu#user-installation) to install the **h2o4gpu** Python package (either at the system level or in a Python virtual envivonment). The easiest thing to do is to `pip install` the stable release `whl` file. To ensure compatibility, the Python package version number should match the R package version number. 
 
-``` r
-if (!require(devtools)) install.packages("devtools")
+The recomended way of installing the R package can is from CRAN using `install.packages("h2o4gpu")`. To install the development version of the **h2o4gpu** R package, you can install directly from GitHub as follows:
+
+```{r, eval = FALSE}
+library(devtools)
 devtools::install_github("h2oai/h2o4gpu", subdir = "src/interface_r")
 ```
 
-To test your installation, try the following example that builds a simple [XGBoost](https://github.com/dmlc/xgboost) random forest classifier:
+
+### Virtual environments
+
+Using a Python [virtual environment](https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments) is a good solution if you don't want to upgrade your main Python installation to 3.6.  If you installed the **h2o4gpu** Python module into a virtual environment, you will have to add a line of code to tell R which Python envivonment you want to use:
+
+```{r, eval = FALSE}
+library(reticulate)
+use_virtualenv("/home/username/venv/h2o4gpu")  # set this to the path of your venv
+```
+If you have installed the package using Anaconda, then you can try to use the `use_condaenv()` function (though we have not tested this yet).
+
+### Test installation
+
+To test your installation, try the following example that builds a simple random forest classifier:
 
 ``` r
```

Comment:
we didn't have `eval=FALSE` here earlier? I wonder why this hasn't error-ed out on CRAN.

Reply:
maybe there's global default for `eval=FALSE` already.

Reply:
This is just the README.md, not the vignette, so we don't need `eval = FALSE`.

Reply:
Oops sorry. I better stop coding today. Please ping me when you are done with changes. I'll take a look another later.